### ISSUE TITLE
a lot of the dlc's are still locked out.

### ISSUE THREAD

**Issue by lucifer8843** [8fn/DLC-Unlocker-PD2#18]
forgive me if im being dumb first time doing this but i put the dlc unlocker in my mods folder and it did unlock most weapons but the outfits, some weapons, and characters are still locked out am i missing something?

**Comment by usergeneratedrandomness** [comment]
yeah some attachments, skins are out. Also is there no way to get scarface?

**Comment by jlsuekko** [comment]
hey i have payday 2 on epic game how can i get all free dlc
